compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmMethodGen.java

```diff
@@ -1446,6 +1454,8 @@ public void genJMethodForBFunc(BIRFunction func,
 
         String currentPackageName = getPackageName(module.org.value, module.name.value, module.version.value);
         BIRVarToJVMIndexMap indexMap = new BIRVarToJVMIndexMap();
+        String initClass = getModuleLevelClassName(module.org.value, module.name.value, module.version.value,
+                MODULE_INIT_CLASS_NAME);
```

Comment:
This statement can be moved inside the body of the [if condition](https://github.com/ballerina-platform/ballerina-lang/pull/23505/files#diff-71f3d02468e0a591b2345d3dd6a00c18R1489) added below as initClass will only be used when that if condition becomes true right?